src/audiovideofacade/DefaultAudioVideoFacade.ts

```diff
@@ -251,8 +251,17 @@ export default class DefaultAudioVideoFacade implements AudioVideoFacade {
     this.trace('realtimeSubscribeToVolumeIndicator', attendeeId);
   }
 
-  realtimeUnsubscribeFromVolumeIndicator(attendeeId: string): void {
-    this.realtimeController.realtimeUnsubscribeFromVolumeIndicator(attendeeId);
+  realtimeUnsubscribeFromVolumeIndicator(
+    attendeeId: string,
+    callback?: (
+      attendeeId: string,
+      volume: number | null,
+      muted: boolean | null,
+      signalStrength: number | null,
+      externalUserId?: string
+    ) => void
```

Comment:
It would be nice if this type signature were extracted and shared between the sub/unsub functions, here and in the realtime controller. It's now repeated in four places.